renderdoc/serialise/serialiser.h

```diff
@@ -1407,6 +1407,73 @@ class Serialiser
     SerialiseValue(type, byteSize, (char *&)el);
   }
 
+  template <typename U>
+  void SerialiseValue(SDBasic type, size_t byteSize, rdcarray<U> &el)
+  {
+    RDCASSERT(type == SDBasic::Array);
```

Comment:
Rather than this runtime assert, just to be super clear about the purpose of this function I think it should be renamed to `SerialiseArrayValue`.